GMS/VectorSetRDD.py

```diff
@@ -0,0 +1,111 @@
+from __future__ import annotations
+from typing import List
+from pyspark import RDD, SparkContext
+from Set import Set
+from pyspark.sql import SparkSession
+
+
+class VectorSetRDD(Set):
+
+    spark = SparkSession.builder.getOrCreate()
+    sc = spark.sparkContext
```

Comment:
Tbh I'd add it as constructor parameter.

Reply:
This way we will not implement Abstract class/interface